proposals/semi-auto-properties.md

```diff
@@ -47,7 +47,7 @@ The following changes are to be made to [classes.md](https://github.com/dotnet/c
 +
 + If the auto-property has only a semicolon-only get accessor, the backing field is considered `readonly` ([Readonly fields](classes.md#readonly-fields)).
 +
-+ An auto-property is not allowed to only have a single semicolon-only `set` accessor without a `get` accessor.
++ An auto-property must have a `get` accessor, and can optionally have a `set` accessor.
```

Comment:
Either I am missing something, or the original wording looks right to me. I.e. The following code should be fine:
```
int P
{
    set { field+=value; if (field > 100) throw null;}
}
``` 

The original wording could be slightly adjusted without changing its meaning: "An auto-property is not allowed to only have a semicolon-only `set` accessor without a `get` accessor."

Reply:
@AlekseyTs That's exactly it. The discussion is about whether this piece of code will be allowed or not.

Reply:
Wouldn't it seem odd not to allow it? Is there a benefit in making it asymmetrical with `get` accessors this way?

Reply:
It's more just preserving what rules we had before. And keeping symmetry with the existing constructs.

Reply:
> It's more just preserving what rules we had before. And keeping symmetry with the existing constructs.

The rules we had before made sense at that time because an auto-property property like that couldn't do anything useful. That is not the case any more. So the wording prior to this PR actually preserves the old rule in the new reality. I do not think there is a requirement to preserve the old rule verbatim.